reactor-core/src/main/java/reactor/core/scheduler/ElasticScheduler.java

```diff
@@ -190,12 +210,13 @@ void eviction() {
 			if (e.expireMillis < now) {
 				if (cache.remove(e)) {
 					e.cached.exec.shutdownNow();
+					//TODO shouldn't we remove from all at some point?
```

Comment:
@smaldini had a doubt on this part of the implementation while reviewing my changes

Reply:
Yep